torch/_weights_only_unpickler.py

```diff
@@ -335,7 +345,7 @@ def load(self):
             # First and last deserializer ops
             elif key[0] == PROTO[0]:
                 # Read and ignore proto version
-                read(1)[0]
+                self.proto = read(1)[0]
```

Comment:
Let's nicely error (or warn if it sometimes work) for != 2 and suggest they open an issue for adding support.